spec/bolt/outputter/human_spec.rb

```diff
@@ -130,65 +130,10 @@
               end
 
     outputter.print_task_info(Bolt::Task.new(name, metadata, files))
-    expect(output.string).to eq(<<~TASK_OUTPUT)
-
-      cinnamon_roll - A delicious sweet bun
-
-      USAGE:
-      #{command}
-
-      PARAMETERS:
-      - icing: Cream cheese
-          Rich, tangy, sweet
-
-      MODULE:
-      /path/to/cinnamony/goodness
-    TASK_OUTPUT
-  end
-
-  it 'converts Data (undef) to Any' do
```

Comment:
I'm not really sure this test was doing what it claimed to be doing... unless I'm misunderstanding it?

Reply:
This was added when we first added the `task show <task>` subcommand to ensure the formatting was correct - I think that replacing the heredoc string with the new string makes sense, to make sure format doesn't drift, rather than checking individual regexes (for example, adding unnecessary newlines to output wouldn't be caught the new test but would be caught with the old one).

Reply:
Ack sorry, comment applies to the bit *above*. *That* test does seem useless now :sweat_smile: https://github.com/puppetlabs/bolt/commit/e3c61a2a64440ebdcd2dfbc6368d2c730d0cc505 was the original, but it looks like we don't coerce the datatype anymore.

Reply:
Ok, *actually* actually it seems like that functionality should be there and has gotten lost - I think the test was updated erroneously, when it was really catching a bug. I'll make a ticket and we can discuss - I think you can remove the test for now.